src/pages/LikedFoodPage.js

```diff
@@ -29,12 +30,15 @@ export default function LikedFoodPage() {
           </li>
         </ListStyle>
       </main>
+      <footer>
+        <Navigation data-testid={'navtest'} />
+      </footer>
     </LikedFoodPageStyle>
   );
 }
 
 const LikedFoodPageStyle = styled.section`
-  background-color: var(--peach);
+  background-color: var(--white);
   display: grid;
   grid-template-rows: 10vm 1fr 50px;
```

Comment:
What does vm stand for? vw viewport width?

Reply:
oh, typo, it should be `vmin`, thanks a lot